commands/channel_users_e2e_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/api4"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestChannelUsersAddCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.th.CreateUser()
+	user, appErr := s.th.App.CreateUser(&model.User{Email: s.th.GenerateTestEmail(), Username: model.NewId(), Password: model.NewId()})
+	s.Require().Nil(appErr)
+
+	_, appErr = s.th.App.AddUserToTeam(s.th.BasicTeam.Id, user.Id, "")
+	s.Require().Nil(appErr)
+
+	channelName := api4.GenerateTestChannelName()
+	channel, appErr := s.th.App.CreateChannel(&model.Channel{
+		TeamId:      s.th.BasicTeam.Id,
+		Name:        channelName,
+		DisplayName: "db_" + channelName,
```

Comment:
What does `db_` mean here?

Reply:
Typo, it was meant to be "dn" - short for display name. This approach is used in mattermost-server tests. Fixed.